gulpfile.js

```diff
@@ -59,31 +85,30 @@ gulp.task('package-resources', ['default'], () => {
   ]);
 });
 
-const browserifyCb = (cb) => {
-  return (err, stdout, stderr) => {
-    console.log(stdout);
-    console.error(stderr);
-    cb(err);
-  };
-};
-
-gulp.task('package-render', ['default'], (cb) => {
-  exec('browserify --node -e build/index.js -o ./package/index.js --im', browserifyCb(cb));
+gulp.task('package-render', ['default', 'package-init'], () => {
+  browserify('build/index.js', browserifyOpts)
+    .bundle()
+    .pipe(fs.createWriteStream('package/index.js'));
 });
 
-gulp.task('package-main', ['default'], (cb) => {
-  exec('browserify --node --igv=false -e build/electron-app.js -o ./package/electron-app.js --im', browserifyCb(cb));
+gulp.task('package-main', ['default', 'package-init'], () => {
+  browserify('build/electron-app.js', browserifyOpts)
+    .bundle()
+    .pipe(fs.createWriteStream('package/electron-app.js'));
```

Comment:
Yeah, because otherwise I need to use `vinyl-transform` or whatever.
At the same time, since I'm forced to use `mkdirp` because of the way `fs.createWriteStream()` works, I might as well use `vinyl`